spec/workflow/client_spec.rb

```diff
@@ -756,13 +756,38 @@
     end
 
     let(:url) { 'dor/objects/druid:123/versionClose' }
-    it 'calls the versionClose endpoint with druid' do
-      client.close_version(@repo, @druid)
+
+    context 'with positional arguments' do
+      before do
+        allow(Deprecation).to receive(:warn)
+      end
+
+      it 'calls the versionClose endpoint with druid' do
+        client.close_version(@repo, @druid)
+        expect(Deprecation).to have_received(:warn)
+      end
+
+      it 'optionally prevents creation of accessionWF' do
+        expect(mock_http_connection).to receive(:post).with('dor/objects/druid:123/versionClose?create-accession=false').and_call_original
+        client.close_version(@repo, @druid, false)
+        expect(Deprecation).to have_received(:warn)
+      end
     end
 
-    it 'optionally prevents creation of accessionWF' do
-      expect(mock_http_connection).to receive(:post).with('dor/objects/druid:123/versionClose?create-accession=false').and_call_original
-      client.close_version(@repo, @druid, false)
+    context 'with kwargs' do
+      it 'calls the versionClose endpoint with druid' do
```

Comment:
is this `it` block missing an expectation?

Reply:
this is copying the test from https://github.com/sul-dlss/dor-workflow-client/pull/142/files#diff-dde6862df2d2e0979838903a65c0a096L760, which didn't have an expectation, and duplicating it with kwargs.  Presumably this test depends on only the stubbed paths being called.